artifacts_test.py

```diff
@@ -121,23 +122,27 @@ def write_back_cache(self) -> int | None:
         return None
 
     def check_cluster_name(self):
+        self.actions_log.info("Checking cluster name", target=self.node.name)
         with self.node.remote_scylla_yaml() as scylla_yaml:
             yaml_cluster_name = scylla_yaml.cluster_name
 
         self.assertTrue(self.db_cluster.name == yaml_cluster_name,
                         f"Cluster name is not as expected. Cluster name in scylla.yaml: {yaml_cluster_name}. "
                         f"Cluster name: {self.db_cluster.name}")
+        self.actions_log.info("Cluster name is validated", target=self.node.name)
 
     def run_cassandra_stress(self, args: str):
         stress_cmd = f"{self.node.add_install_prefix(STRESS_CMD)} {args} -node {self.node.ip_address}"
         if self.params.get('client_encrypt'):
             transport_str = c_s_transport_str(
                 self.params.get('peer_verification'), self.params.get('client_encrypt_mtls'))
             stress_cmd += f" -transport '{transport_str}'"
-
+        self.actions_log.info("Running Cassandra stress", metadata={"stress_cmd": stress_cmd}, target=self.node.name)
```

Comment:
In my opinion, the usage of this is ideal for a context manager. It doesnt need to be used in all cases, but for this one I think it would be ideal